A/algoim/build_tarballs.jl

```diff
@@ -0,0 +1,22 @@
+using BinaryBuilder
+
+name = "algoim"
+version = v"0.1.0"
+sources = [
+    GitSource("https://github.com/algoim/algoim.git", "979cb3b7860b53751291352a929754d525b7fde1"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/
+install_license ${WORKSPACE}/srcdir/algoim/LICENSE
+mkdir -p "${includedir}"
+cp -vr algoim/algoim/* "${includedir}/."
```

Comment:
Judging by the [documentation](https://algoim.github.io/), I think you should put the directory `algoim` itself in `${includedir}`, not its individual files directly:
```suggestion
cd ${WORKSPACE}/srcdir/algoim
install_license LICENSE
mkdir -p "${includedir}"
cp -vr algoim "${includedir}/."
```